src/main/java/com/manardenza/dao/AbstractDao.java

```diff
@@ -3,14 +3,15 @@
 import com.manardenza.entity.AbstractObject;
 import com.manardenza.entity.Hotel;
 import com.manardenza.entity.User;
+import com.manardenza.utils.DatabaseSeeder;
 
 import java.util.ArrayList;
 import java.util.List;
 
-public abstract class AbstractDao<T extends AbstractObject> {
+public abstract class AbstractDao<T extends AbstractObject> extends DatabaseSeeder {
 
-    protected List<Hotel> hotelList = new ArrayList<>();
-    protected List<User> userlList = new ArrayList<>();
+    protected List<Hotel> hotelList = hotelsSeederList;
```

Comment:
You shouldn't automatically fill all DAOs with seed data upon starting the program. Do a separate public void method called seedDatabase() which will perform this step. Then you will be able to manually call this method when needed (initially upon launching main())